assessments/guide/outline.md

```diff
@@ -1,88 +1,179 @@
-* Security Degradation.  A discussion about the resulting security when various attacks are launched.  Note, that no system is secure in all scenarios, hence it is expected that this will include areas where attacks compromise all meaningful security.  (e.g., If an attacker is able to compromise the “master” Flibble server, they may read, write, or delete any content stored on any system)
-* Compensating Mechanisms.  Additional architectural decisions, configuration settings, options, etc. designed to reduce overall attack vector and success (minimize impact) 
+* Attacker Motivations.  A discussion about the likely goals of an attacker as
+  well as the kind of attacker (do not forget to include discussion of insider
+threat with trusted access to the project).  This likely relates closely to the
+impact of different attacks in the scenarios.  (e.g., In the password hash case,
+the attacker wants to expose those hashes on the Flibble server.  However, a
+Flibble cloudlet attacker may find it more interesting to bring down the
+service.)
+* Predisposing Conditions. A list of potential vulnerabilities and
+  configurations of the project that could potentially be exploited or used
+correctly to result in an increased likelihood of attack success. Include any
+trust relationships with other projects that pose a risk of compromise for this
+project (i.e. compromise of the LDAP results in loss of access control integrity
+for the project)
+* Expected Attacker Capabilities.  A description of likely capabilities that the
+  attacker has in these scenarios should be described.  Both assumptions about
+the strength and limitations of attackers should be described (e.g., We assume
+that an attacker may be able to exploit implementation errors in some set of the
+servers to take control of them.  However, we assume the attacker cannot break
+AES or SHA256.)
+* Attack Risks and Effects.  A rough estimation of the risk posed by different
+  attacks, and potential negative consequences (e.g., The master Flibble server
+only communicates with Flibble servers using a minimalistic API that is formally
+verified and written in Rust.)
+* Security Degradation.  A discussion about the resulting security when various
+  attacks are launched.  Note, that no system is secure in all scenarios, hence
+it is expected that this will include areas where attacks compromise all
+meaningful security.  (e.g., If an attacker is able to compromise the “master”
+Flibble server, they may read, write, or delete any content stored on any
+system)
+* Compensating Mechanisms.  Additional architectural decisions, configuration
+  settings, options, etc. designed to reduce overall attack vector and success
+(minimize impact) 
+* Inclusion of a threat model if one exists is encouraged.
 
```

Comment:
We need to talk more here about the blast radius (likely collateral impact from an attack).